EGO Forum Enhancement.user.js

```diff
@@ -288,6 +319,71 @@ function handleBanReport() {
+    // Variable gets dereferenced - reference it again
+    explain = body.querySelector(':nth-child(2) > dd > textarea');
+    explain.style.height = '200px';
+    explain.setAttribute('maxlength', '1024');
+    var div = body.querySelector(':nth-child(4) > dd > div > .formSubmitRow-controls');
+
+    addPreset("No MAUL account", div, function() {
+        reason.value = "MAUL account must be created and verified";
+        explain.value = "In order for you to fix this you'll need to click the MAUL link at the top of the page in the navbar, click \"Edit Game IDs,\" \
+then click the Sign in through Steam button under the Source ID section. Once you've done so, please reply to this post!"
+    })
+
+    addPreset("Steam Verification", div, function() {
+        reason.value = "Steam account must be verified in MAUL";
+        explain.value = "In order for you to fix this you'll need to click the MAUL link at the top of the page in the navbar, click \"Edit Game IDs,\" \
+then click the Sign in through Steam button under the Source ID section. Once you've done so, please reply to this post!"
+    })
+
+    addPreset("Minecraft Verification", div, function() {
+        reason.value = "Minecraft ID must be verified in MAUL";
+        explain.value = "In order for you to fix this you'll need to click the MAUL link at the top of the page in the navbar, click \"Edit Game IDs,\" \
+then under ID for Minecraft, input your Minecraft username, click Convert to Game ID, then log onto our Minecraft server. Once you've done so, please reply to this post!"
+    })
+
+    addPreset("Battlefield Verification", div, function() {
+        reason.value = "Battlefield account must be verified in MAUL";
+        explain.value = "In order for you to fix this you'll need to click the MAUL link at the top of the page in the navbar, in MAUL hover over the home link in the top left, \
+click help, then follow the instructions for Battlefield. Once you have done so, please reply to this post!";
+    })
+
+    addPreset("Discord Verification", div, function() {
+        reason.value = "Discord ID must be verfied in MAUL";
+        explain.value = "In order for you to fix this you'll need to click the MAUL link at the top of the page in the navbar, click \"Edit Game IDs,\" \
+then click the sign in through Discord button under the discord ID section. Once you have done so, please reply to this post!"
+    })
+
+    addPreset("Inappropriate Name", div, function() {
+        reason.value = "Inappropriate Name";
+        explain.value = "As for your name, Please click [URL='https://www.edgegamers.com/account/username']here[/URL] and fill out a name change request. \
+After you fill it out, please wait while your name change request is finalized and the change is completed. \
+Once it is done your application process will resume. If you want to have an understanding on our naming policy inside of eGO please click [URL='https://www.edgegamers.com/threads/378540/']here[/URL].";
```

Comment:
Could we modify these to be more in line with indenting, potentially swapping out `\` for `\n` if needed?

Reply:
There's no easy way to accomplish this to my knowledge, maybe somebody else can figure this out.